src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $border;
+  padding: 10px;
+  max-width: 7 * $cellSize + 6 * $border;
```

Comment:
maybe max-width should contain paddings?

Reply:
done